web-app/js/portal/details/StylePanel.js

```diff
@@ -152,6 +152,11 @@ Portal.details.StylePanel = Ext.extend(Ext.Container, {
         this.legendImage.show();
 
         this.styleCombo.setValue(styleName);
+
+        if (scaleRange) {
+            var scaleRangeString = toNSigFigs(scaleRange.min, 2) + "," + toNSigFigs(scaleRange.max, 2);
+            trackLayerControlUsage('layerControlTrackingActionStyleRange', scaleRangeString, this.dataCollection.getTitle());
+        }
```

Comment:
This seems to me to be in the wrong place. It should just go in the NcWmsScaleRangeControls.applyNewScale() method.